packages/web-react/src/components/CheckboxField/README.md

```diff
@@ -5,24 +5,24 @@ and an optional message. It could be disabled or have an error state. The label
 and show if the input is required.
 
 ```jsx
-<CheckboxField id="example" name="example" isRequired isChecked validationState="error" message="validation failed" />
+<CheckboxField id="example" name="example" isRequired isChecked validationState="danger" message="validation failed" />
 ```
 
 ## Available props
 
-| Name              | Type    | Description                    |
-| ----------------- | ------- | ------------------------------ |
-| `id`              | string  | Input and label identification |
-| `name`            | string  | Input name                     |
-| `label`           | string  | Label text                     |
-| `value`           | string  | Input value                    |
-| `message`         | string  | Validation or help message     |
-| `validationState` | `error` | Type of validation state       |
-| `isDisabled`      | boolean | Whether is field disabled      |
-| `isItem`          | boolean | To render in [Item][item] mode |
-| `isRequired`      | boolean | Whether is field required      |
-| `isChecked`       | boolean | Whether is field checked       |
-| `isLabelHidden`   | boolean | Whether is label hidden        |
+| Name              | Type     | Description                    |
```

Comment:
can you please update this table so it matches other API tables? (formatting, missing default and required columns)

Reply:
Not one table with props was correct. I remodeled and added to all of them.